todo.js

```diff
@@ -0,0 +1,59 @@
+const addTaskBtn = document.getElementById("addTask");
+const taskInputText = document.getElementById("taskInput");
+const taskList = document.getElementById("taskList");
+const errorMessage = document.getElementById("errorMessage");
+
+let todos = [];
+
+function generateUniqueId() {
+  return Date.now();
+}
+const resetTaskInput = () => {
+  taskInputText.value = "";
+};
+const isUserInputValid = (textContent) => {
+  return textContent.trim() !== "";
+};
+
+const validateInput = () => {
+  if (isUserInputValid(taskInputText.value)) {
+    errorMessage.innerHTML = "";
+    return true;
+  } else {
+    errorMessage.innerHTML = "Please add a task";
+    return false;
+  }
```

Comment:
Early return is good convention that we often try to follow to avoid `else` block

```suggestion
  if (isUserInputValid(taskInputText.value)) {
    errorMessage.innerHTML = "";
    return true;
  }
  
  errorMessage.innerHTML = "Please add a task";
  return false;
```